### PR TITLE
Joyce send key

### PR DESCRIPTION
**Pull request by joycechen612**
please help review . thanks

### CODE REVIEW

**Review by Wei1150 — DISMISSED** [review]
LGTM

**Review by WenjiZ — DISMISSED** [review]

**Comment by ShaolingStouder** [comment]
LGTM!

**Review by ShaolingStouder — DISMISSED** [review]

**Review by minyizeng — DISMISSED** [review]
Good job

File: src/test/java/utilities/Util.java [hunk 1]
```diff
@@ -12,5 +16,19 @@ public static void wait(int seconds) {
 			e.printStackTrace();
 		}
 	}
+	
+	/*
+	 * sendKey function
+	 * Author:Joyce
+	 */
+	public static void sendKey(WebDriver driver,String value,String xpathLocator) {
+		WebElement ele = driver.findElement(By.xpath(xpathLocator));
```
Comment (boratech-macmini) [inline src/test/java/utilities/Util.java]: Instead of passing in xpathLocator, pass in a `By` object would make it more reusable

**Review by boratech-macmini — CHANGES_REQUESTED** [review]
Formatting is off

**Review by minyizeng — DISMISSED** [review]

**Pull request closed without merging**